Makefile

```diff
@@ -9,7 +9,7 @@ NO_COLOR=\\e[39m
 OK_COLOR=\\e[32m
 ERROR_COLOR=\\e[31m
 WARN_COLOR=\\e[33m
-PORT=8889
```

Comment:
Is there any special reason for changing the ports? It's ok anyways, just asking out of curiosity

Reply:
No, port 8889 was just already in use on my computer :)